.github/workflows/ci_testing.yml

```diff
@@ -92,7 +92,7 @@ jobs:
           cmake --build . --target Docs --config Debug
 
   Build-and-Test-Win:
-    runs-on: windows-2019
+    runs-on: windows-2020
```

Comment:
@chkim-usgs Reading the blog post about it, we should switch to 2022 or 2025. I would suggest 2025 to get more time before we have to do this again. You might also have to change the visual studio version but just try setting the run-on to windows-2025